gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -483,42 +521,34 @@ def run(argv=None):
             table_name,
             sharding.get_output_table_total_base_pairs(i),
             schema_file)
-      _ = (variants[i] | 'VariantToBigQuery' + table_suffix >>
-           variant_to_bigquery.VariantToBigQuery(
-               table_name,
-               schema,
-               append=known_args.append,
-               update_schema_on_append=known_args.update_schema_on_append,
-               allow_incompatible_records=known_args.allow_incompatible_records,
-               omit_empty_sample_calls=known_args.omit_empty_sample_calls,
-               num_bigquery_write_shards=known_args.num_bigquery_write_shards,
-               null_numeric_value_replacement=(
-                   known_args.null_numeric_value_replacement)))
-
-  if known_args.output_avro_path:
-    # TODO(bashir2): Add an integration test that outputs to Avro files and
-    # also imports to BigQuery. Then import those Avro outputs using the bq
-    # tool and verify that the two tables are identical.
-    for i in range(num_shards):
-      avro_path = sample_info_table_schema_generator.compose_table_name(
-          known_args.output_avro_path, sharding.get_output_table_suffix(i))
-      _ = (
-          variants[i] | 'VariantToAvro' >>
-          variant_to_avro.VariantToAvroFiles(
-              avro_path,
-              header_fields,
-              processed_variant_factory,
-              variant_merger=variant_merger,
-              allow_incompatible_records=known_args.allow_incompatible_records,
-              omit_empty_sample_calls=known_args.omit_empty_sample_calls,
-              null_numeric_value_replacement=(
-                  known_args.null_numeric_value_replacement))
-      )
-
-  result = pipeline.run()
-  result.wait_until_finish()
-
-  metrics_util.log_all_counters(result)
+      pipeline_common.load_avro_to_output_table(table_name,
```

Comment:
Hmm, does it make sense to multi-thread this to parallelize BQ load? If BQ doesn't complain I think we should.

Edit: Hmmm, now that I read a bit more seems like handling of deleting tables if Exceptions is raised could be problematic (but yet, not impossible). If time delay is not too too bad, maybe it's ok to keep this... I'm open to discussion.